params/config.go

```diff
@@ -292,20 +292,28 @@ type NodeConfig struct {
 	// discoverable peers with the discovery limits.
 	RequireTopics map[discv5.Topic]Limits `json:"RequireTopics"`
 
-	// StatusServiceEnabled enables status service api
-	StatusServiceEnabled bool
-
-	// InstallationId id of the current installation
-	InstallationID string
-
-	// DebugAPIEnabled enables debug api
-	DebugAPIEnabled bool
-
 	// MailServerRegistryAddress is the MailServerRegistry contract address
 	MailServerRegistryAddress string
+}
 
+// StatusServiceConfig defines options used by shhext service.
+type StatusServiceConfig struct {
+	// Enabled should be true if we want to use status service.
+	Enabled bool
+	// BackupDisabledDataDir is the file system folder the node should use for any data storage needs that it doesn't want backed up.
+	BackupDisabledDataDir string `validate:"required"`
+	// InstallationId id of the current installation
+	InstallationID string
+	Debug          bool
```

Comment:
I would keep it named `DebugAPIEnabled`. `Debug` even in `StatusServiceConfig` is not descriptive enough.